app/controllers/admin/playlists_controller.rb

```diff
@@ -7,5 +7,13 @@ def authorize_resource
       @resource = Student.find(params[:student_id]).playlist
       authorize(resource)
     end
+
+    def collection
+      Playlist.includes(:student)
+              .where(users: {grove_id: current_user.grove.id})
+              .where('name ILIKE ?', "%#{params[:student_search]}%")
```

Comment:
does this work when the search is blank? seems like you might want to default the collection to something, maybe? Also, I think you want to probably wildcard the `ILIKE` unless you don't care about partial matches.